pkg/pgmodel/metrics/database/metrics.go

```diff
@@ -59,41 +77,127 @@ var metrics = []metricQueryWrap{
 		isHealthCheck: true,
 	},
 	{
-		metric: prometheus.NewGauge(
+		metrics: gauges(
 			prometheus.GaugeOpts{
 				Namespace: util.PromNamespace,
 				Subsystem: "sql_database",
 				Name:      "chunks_count",
 				Help:      "Total number of chunks in TimescaleDB currently.",
 			},
-		),
-		// Compressed_chunk_id is null for both yet to be compressed and already compressed chunks.
-		query: `select count(*)::bigint from _timescaledb_catalog.chunk where dropped=false and compressed_chunk_id is null`,
-	},
-	{
-		metric: prometheus.NewGauge(
 			prometheus.GaugeOpts{
 				Namespace: util.PromNamespace,
 				Subsystem: "sql_database",
 				Name:      "chunks_created",
 				Help:      "Total number of chunks created since creation of database.",
 			},
-		),
-		// Compressed_chunk_id is null for both yet to be compressed and already compressed chunks.
-		query: `select count(*)::bigint from _timescaledb_catalog.chunk where compressed_chunk_id is null`,
-	},
-	{
-		metric: prometheus.NewGauge(
 			prometheus.GaugeOpts{
 				Namespace: util.PromNamespace,
 				Subsystem: "sql_database",
 				Name:      "chunks_compressed_count",
 				Help:      "Total number of compressed chunks in TimescaleDB currently.",
 			},
 		),
-		query: `select count(*)::bigint from _timescaledb_catalog.chunk where dropped=false and compressed_chunk_id is not null`,
+		// Compressed_chunk_id is null for both yet to be compressed and already compressed chunks.
+		query: `SELECT 
+				count(*) FILTER (WHERE dropped=false AND compressed_chunk_id IS NULL)::BIGINT AS chunks_count,
+				count(*) FILTER (WHERE compressed_chunk_id IS NULL)::BIGINT AS chunks_created,
```

Comment:
while not part of the changes introduced in this pr. I just noticed that chunks_created is a pretty meaningless metric. The problem is that in timescaledb chunks are sometimes dropped by just deleting the row and sometimes by updating dropped=false (depends on how/if cont aggs are defined) so just relying on dropped=true is misleading. Can we just drop this metric? cc @Harkishen-Singh .

Reply:
Sorry, I do not understand the problem. The aim of `chunks_created` is to keep track of all chunks created in TimescaleDB since starting and it is meaningful for the user.

This is the query for knowing `promcsale_sql_database_chunks_created` is `select count(*)::bigint from _timescaledb_catalog.chunk where compressed_chunk_id is null` and I don't see where is `dropped=true` here.

Reply:
> in timescaledb chunks are sometimes dropped by just deleting the row and sometimes by updating dropped=false (depends on how/if cont aggs are defined) so just relying on dropped=true

Which `row`? Sorry, but the above statement is confusing to me.

Reply:
The _timescaledb_catalog.chunk can be deleted. which would then cause chunks_created to decrease according to the current query. That would be weird, no?

Reply:
Yes that will be weird. In that case chunks_created metric would be wrong and we should remove it.

I guess, this behaviour is for performance reasons, like preventing the chunks table from getting too big, but at the same time, this is a buggy behaviour from db, right!